Java/corenative/src/main/java/com/breadwallet/corenative/crypto/BRCryptoTransferState.java

```diff
@@ -30,27 +33,39 @@
             public long transactionIndex;
             public long timestamp;
             public BRCryptoFeeBasis feeBasis;
+            public int success;
+            public byte[] error = new byte[1];
```

Comment:
Isn't this (CRYPTO_TRANSFER_INCLUDED_ERROR_SIZE + 1)?

Reply:
Yes, but I don't think it matters.  The constructor doesn't actually copying into that; it just assigns the fully-sized argument for `error`.

Reply:
The JNA docs indicate that the size of the structure is calculated based off of this, when you have an array within a structure.

Reply:
In https://java-native-access.github.io/jna/4.2.1/overview-summary.html#structures, search for:
Structures with nested arrays require an explicit constructor to ensure the structure size is properly calculated.

Reply:
It seems the code does not go through the 'constructor' `public included_struct(...)` - so, in fact, I am only getting the first character of the error.